plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/xsd/com.google.cloud.tools.eclipse.appengine.libraries.xsd

```diff
@@ -202,6 +205,51 @@ unspecified, the javadoc will be downloaded from the artifact location if availa
       </xs:complexType>
    </xs:element>
 
+   <xs:element name="enablement">
+      <xs:annotation>
+         <xs:documentation>
```

Comment:
There's a schema to define the enablement language.
`<include schemaLocation="schema://org.eclipse.core.expressions/schema/expressionLanguage.exsd"/>
`